src/js/containers/Buttons.js

```diff
@@ -138,7 +138,7 @@ const mapStateToProps = (state) => {
             iconProps: iconProps
         })
     }
-    if (softButtons.length > 1) {
+    if (softButtons.length > 1 && subscribedButtons.CUSTOM_BUTTON === true) {
```

Comment:
```suggestion
    if (softButtons.length > 1) {
```
I don't think that we should be preventing these buttons from appearing, Core will keep track of whether or not the app is subscribed.

Reply:
@jacobkeeler please find fix [here](https://github.com/LuxoftSDL/generic_hmi/pull/26/commits/c1a08c356f3f507001a80145f48d31b33c44b097).